backend/telemetry_core/src/aggregator/inner_loop.rs

```diff
@@ -211,13 +213,15 @@ impl InnerLoop {
         let max_queue_len = self.max_queue_len;
         let (metered_tx, metered_rx) = flume::unbounded();
 
-        // Keep count of the number of messages we drop for the sake of metric reporting
+        // Keep count of the number of dropped/total messages for the sake of metric reporting
         let dropped_messages = Arc::new(AtomicU64::new(0));
+        let total_messages = Arc::new(AtomicU64::new(0));
```

Comment:
Could put it on the same `Arc` with dropped_messages.

Reply:
That's true; I'll do that actually!

Reply:
I had a go at this but decided not to do it in the end just because it made the code messier :)